virt/virt-4-8-release-notes.adoc

```diff
@@ -72,7 +72,7 @@ The SVVP Certification applies to:
 
 //CNV-12269 When a VM frees space on a virtual disk the discard requests are passed to the underlying storage device which may free up real storage capacity.
 
-//CNV-12270 CDI can now automatically choose preferred accessMode and volumeMode settings when importing VM disk images.
+* The Conainerized Data Interface (CDI) now automatically chooses preferred `accessModes` and `volumeMode` settings when you import virtual machine disk images.
```

Comment:
Containerized

We need to be a bit more measured with this note.  None of: the UI, virtctl, or VM Templates are using the 'storage' field of the DataVolume spec to activate this new behavior.  I think we should state that "a new way of specifying Data Volumes is available and, when used, will allow the system to optimize 'accessModes', 'volumeMode' and storage capacity when allocating storage.

Reply:
Hi @aglitke - is it possible to say this:

* You can now specify data volumes using the storage API. The storage API, unlike the PVC API, allows the system to optimize 'accessModes', 'volumeMode' and storage capacity when allocating storage.

I understand some of the generalization but I think mentioning the storage API is important. WDYT?